graphics/tinygl/zbuffer.cpp

```diff
@@ -139,30 +139,65 @@ void FrameBuffer::delOffscreenBuffer(Buffer *buf) {
 
 void FrameBuffer::clear(int clearZ, int z, int clearColor, int r, int g, int b) {
```

Comment:
Is it clearer now ? Or is it the way I use the `for` loop which is surprising ?

Also, thanks for making me take a 3rd look at this code, as I could simplify `clear` a bit and discovered `clearRegion` was broken.